winml/adapter/winml_adapter_c_api.h

```diff
@@ -0,0 +1,469 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Can you document here or at an appropriate place that these APIs are an implementation detail to support the WinML APIs only and not intended for direct usage and that any direct usage of these APIs is not supported by the product? Just want to ensure any usage of these APIs don't turn into any support issues for us.

Reply:
done